src/core/abr/abr_manager.ts

```diff
@@ -148,3 +151,38 @@ export default class ABRManager {
     return originalBandwidthEstimator;
   }
 }
+
+/**
+ * Create Observable that merge several throttling Observables into one.
+ * @param {Observable} limitWidth$ - Emit the width at which the chosen
+ * Representation should be limited.
+ * @param {Observable} throttleBitrate$ - Emit the maximum bitrate authorized.
+ * @param {Observable} throttle$ - Also emit the maximum bitrate authorized.
+ * Here for legacy reasons.
+ * @returns {Observable}
+ */
+function createFilters(
```

Comment:
I moved the `create_filters.ts` file here, because there was too much files everywhere in `src/core/abr` and thus it became difficult to dive into